src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  hasLoadingError: boolean,
+  setHasLoadingError: (arg: boolean) => void,
+  isAddingErrorShown: boolean,
+  setIsAddingErrorShown: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const ErrorNotification = ({
+  hasLoadingError,
+  setHasLoadingError,
+  isAddingErrorShown,
+  setIsAddingErrorShown,
+}: Props) => {
```

Comment:
```suggestion
export const ErrorNotification: FC<Props> = ({
  hasLoadingError,
  setHasLoadingError,
  isAddingErrorShown,
  setIsAddingErrorShown,
}) => {
```
When you type your props right here you don't have access to children.